semcore/carousel/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [3.29.0] - 2024-03-14
+
+### Added
+
+- `keyboardFocusEnhance` to Carousel component.
+- `tabIndex = -1` to Buttons in Carousel.
```

Comment:
Let's just say that you made the whole carousel focusable and prev/next button non focusable

Maybe also we need to mention changed arrows navigation?

Reply:
fixed